client/new-stack-editor/lib/styl/main.styl

```diff
@@ -1,7 +1,181 @@
+
+// FontAwesome
+$fa-font-path = "http://netdna.bootstrapcdn.com/font-awesome/4.7.0/fonts"
+@import "../../../node_modules/font-awesome-stylus/stylus"
+
 @css {
 /*
 New Stack Editor stylesheet
 stackeditor.styl
 */
 }
 
+dark-panel-bg               = #2B2B30
+dark-tab-header-bg          = #131315
+dark-active-tab-color       = #9C9C9C
+dark-tab-color              = #5C5C5C
+
+resize-handle-color         = #6EA3E5
+stack-bg-color              = #232327
+stack-fg-color              = #59595C
```

Comment:
[minor] prefixing variables with `$` makes it easier to distinguish from other ones.